src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx

```diff
@@ -16,70 +14,48 @@ export const IsomerDetailGroupView = ({
   const activePos = editor.state.selection.anchor
   const selected = activePos >= startPos && activePos <= endPos
 
-  const { onDrawerOpen } = useEditorDrawerContext()
   const otherButtons = (
     <>
       <Box
-        display={selected ? "block" : "none"}
+        display={selected ? "flex" : "none"}
         position="absolute"
         width="100%"
+        justifyContent="end"
       >
         <HStack
+          bgColor="interaction.mainLight.default"
+          borderRadius="0.25rem"
+          border="1px solid"
           borderColor="base.divider.medium"
-          display="flex"
-          justifyContent="flex-end"
+          boxShadow="0px 8px 12px 0px rgba(187, 187, 187, 0.50)"
```

Comment:
could we use a token for this?
```ts
boxShadow: `0 0 0 1px var(--chakra-colors-gray-400)`
```

Reply:
@sehyunidaaa may i get token for this please?

Reply:
can use chakra tokens from here https://github.com/chakra-ui/chakra-ui/blob/f65586674cb9241feeacfce7c5672c9129d1704e/packages/theme/src/foundations/colors.ts#L34

Reply:
hmmm can we use this `shadow-sm` from OGPDS? 
https://github.com/opengovsg/design-tokens/blob/09eeb37eaca44f0cb56e64d992cdee75b34e7661/tokens.json#L1109C1-L1110C1 ? might look a bit different but not earth-shattering, it's a small shadow

but we've been using the same bespoke shadow for other bubble menus btw
https://github.com/isomerpages/isomercms-frontend/blob/825766a6fc7aa300ba44a87899b624107c255e31/src/layouts/components/Editor/components/CardsBubbleMenu.tsx#L20